code-challenges.test.js

```diff
@@ -36,14 +78,61 @@ const nums2 = [109, 5, 9, 67, 8, 24]
 
 // b) Create the function that makes the test pass.
 
+// -->pseudo 
+// Create a new variable called arr and tie it to the given const (nums2, nums1)
+// Creat a var for max and min value that arrange the arrays based on greater than or less than.
+// console log for MAX and MIN
+
+ var arr = nums2; nums1
+  var max = arr.reduce(function(x,y){
+     return (x > y) ? x : y;
+  });
+  var min = arr.reduce(function(x,y){
+     return (x < y) ? x : y;
+  });
+  console.log('Max: '+ max);
+  console.log('Min: '+ min);
```

Comment:
Nice approach, however currently this is not set up as a function as the prompt asks.